src/main/java/Duke.java

```diff
@@ -1,10 +1,137 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+    public static class Task {
+        protected String description;
+        protected boolean isDone;
+        protected char taskType;
+
+        public Task(String description) {
+            this.description =  description;
+            this.isDone = false;
+        }
+
+        public String getStatusIcon() {
+            return (isDone ? "X" : " ");
+        }
+        public void markAsDone() {
+            this.isDone = true;
+        }
+
+    }
+
+    static String exitTrigger = "bye";
+    static String listTrigger = "list";
+    static String doneTrigger = "done";
+    static Task[] commands = new Task[100];
+
+    public static void greet() {
+        String logo = "       ___                             | '  \\\n" +
+                "   ___  \\ /  ___         ,'\\_           | .-. \\        /|\n" +
+                "   \\ /  | |,'__ \\  ,'\\_  |   \\          | | | |      ,' |_   /|\n" +
+                " _ | |  | |\\/  \\ \\ |   \\ | |\\_|    _    | |_| |   _ '-. .-',' |_   _\n" +
+                "// | |  | |____| | | |\\_|| |__    //    |     | ,'_`. | | '-. .-',' `. ,'\\_\n" +
+                "\\\\_| |_,' .-, _  | | |   | |\\ \\  //    .| |\\_/ | / \\ || |   | | / |\\  \\|   \\\n" +
+                " `-. .-'| |/ / | | | |   | | \\ \\//     |  |    | | | || |   | | | |_\\ || |\\_|\n" +
+                "   | |  | || \\_| | | |   /_\\  \\ /      | |`    | | | || |   | | | .---'| |\n" +
+                "   | |  | |\\___,_\\ /_\\ _      //       | |     | \\_/ || |   | | | |  /\\| |\n" +
+                "   /_\\  | |           //_____//       .||`  _   `._,' | |   | | \\ `-' /| |\n" +
+                "        /_\\           `------'        \\ |               `.\\  | |  `._,' /_\\\n" +
+                "                                       \\|                    `.\\\n" +
+                "  S. Lu - I solemnly swear that I am up to no good.\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\n"
+                + "What can I do for you?\n");
+    }
+
+    public static void exit() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
```

Comment:
Maybe function name could be call printExitMessage instead?

Reply:
Perhaps a function name called printExitMessage would be better?
```suggestion
    public static void printExitMessage() {
        System.out.println("Bye. Hope to see you again soon!");
    }
```